twitchio/ext/eventsub/server.py

```diff
@@ -234,15 +234,20 @@ async def _callback(self, request: web.Request) -> web.Response:
             logger.warning(f"Unexpected message type: {typ}")
             return web.Response(status=400)
 
-        logger.debug(f"Recived a message type: {typ}")
+        logger.debug(f"Received a message type: {typ}")
         event = _message_types[typ](self, payload, request)
         response = event.verify()
 
+        # Reject messages which do not pass webhook_secret verification!
+        if response.status != 200:
+            return response
+
         if typ == "notification":
-            self.client.run_event(
-                f"eventsub_notification_{models.SubscriptionTypes._name_map[event.subscription.type]}", event
-            )
+            event_name = f"eventsub_notification_{models.SubscriptionTypes._name_map[event.subscription.type]}"
+            logger.info(event_name)
```

Comment:
we dont need a log message every time an event is fired

Reply:
> we dont need a log message every time an event is fired

I find that I need this new log message during development, in order to know what eventsub notifications arrive so that I can implement suitable event handlers for them -- nothing else logs them.

If this is particularly unpalatable to you, another solution that would serve my purpose would be something like a generic "eventsub_notification_arrived" event which supplies the arriving SubscriptionType's name (so that I could log it myself).  But that seems like overkill when just log line is all I really need.